src/main/java/seedu/address/model/person/Faculty.java

```diff
@@ -1,16 +1,24 @@
 package seedu.address.model.person;
 
-// import static java.util.Objects.requireNonNull;
-// import static seedu.address.commons.util.AppUtil.checkArgument;
+import static java.util.Objects.requireNonNull;
+import static seedu.address.commons.util.AppUtil.checkArgument;
 
 /**
  * Represents a Person's faculty in the address book.
  * Guarantees: immutable; is valid as declared in {@link #isValidFaculty(String)}
  */
 public class Faculty {
 
+    /**
+     * The Faculties that are allowed to be used as a valid input.
+     */
+    enum Faculties {
+        SOC, FOS, YLLSOM, FOD, BIZ, SDE, FOE, FOL, YSTCOM, FASS
+
+    }
+
     public static final String MESSAGE_FACULTY_CONSTRAINTS =
-            "Faculty name should be standardized with what NUS uses."
+            "Faculty name should be standardized with what NUS uses. "
```

Comment:
can add into the user guide or in the displayed message the exact values accepted, in case user is unsure :-)

Reply:
Alrighty will do so in the next round of updates!